test_psa_target.py

```diff
@@ -342,11 +348,18 @@ def _get_parser():
 
     parser.add_argument(
         "-b",
-        "--build",
```

Comment:
It is pretty much implied that the option is to `build` the test, than specifying `build_only`, if we are use this, please rename to `--build-only`

Reply:
I'll rework this part

Reply:
Done. Now if `-b`/`--build` and `-r`/`--run` are both passed, it'll build and run just like if neither is passed. It avoids confusion around readability.